### DISCUSSION TITLE
现在有哪些还在维护的协议/配置？

### DISCUSSION THREAD

**Discussion by 517030910405** [v2fly/v2ray-core#540] [Q&A]
我看到过一些使用的例子：
`mKCP+VMESS+动态端口`
`NGINX+WS+TLS+VLESS`
`HTTP+VMESS`
请问这些都是还在维护的主线吗？
有没有 LTS 版本？
谢谢！

**Comment by 517030910405** [comment]
如果一直处于维护状态的 LTS 主线能够在 `README` 里面标出来就方便多了

**Comment by kslr** [comment]
不维护的已经移除了
